### DISCUSSION TITLE
About Traffic Control env

### DISCUSSION THREAD

**Discussion by BackMountainDevil** [opendilab/DI-engine#777] [General]
In the [intro](https://di-engine-docs.readthedocs.io/en/latest/00_intro/index.html#what-is-di-engine) of DI-engine, there are 6 graphs around Environment analysis. One of them is Traffic Control. But I do not find that env in [env example](https://di-engine-docs.readthedocs.io/en/latest/13_envs/index.html)

**Comment by PaParaZz1** [comment]
For this kind of environment, you can refer to our another open-source library [DI-smartcross](https://github.com/opendilab/DI-smartcross).

**Comment by BackMountainDevil** [comment]
cool. still SUMO and CityFlow. Thank you for your link to smartcross

There is a thing named LibSignal, quite like [DI-smartcross](https://github.com/opendilab/DI-smartcross).

- https://arxiv.org/abs/2211.10649
- https://github.com/DaRL-LibSignal/LibSignal

**Reply by PaParaZz1** [reply]
Thanks for your interests. DI-smartcross was mainly updated two years ago, now we don't more plans about this project. BTW, would you like to contribute some new features in DI-smartcross? We can provide the corresponding support if you have some ideas.
